src/shared/assets/sass/v2/blocks/_table.scss

```diff
@@ -46,33 +13,93 @@
     }
 }
 
-.table__header {
-    @include govuk-typography-weight-bold;
-}
-
-.table__context {
-  margin: 0;
-  color: $govuk-secondary-text-colour;
+.truncate {
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
   display: block;
   @include govuk-typography-weight-regular;
-
-
-  // larger screens
-  @media (min-width: 768px) {
-    font-size: 0;
-    text-indent: -1000em;
-  }
+  max-width: 250px;
 }
 
-.table-row:first .table__cell:first-child {
-  border: none;
-  @media (min-width: 768px) {
-    border-bottom: 1px solid #bfc1c3;
+.clickable {
+
+  // If JS is enabled, user can click entire table row to view licence
+  // Hint to user this is possible by displaying hover state on table row
+  .js-enabled & {
+    &:hover {
+      cursor: pointer;
+      background: #f3f2f1;
+    }
   }
 }
 
+.column--15 { width: 15% }
```

Comment:
Can we check if there are standard gov-uk classes that would help with this and the class below?
We should always avoid adding custom CSS wherever possible